api/tools/generate_listeners_test.py

```diff
@@ -1,6 +1,9 @@
 """Tests for generate_listeners."""
 
 import os
+import sys
+
+sys.path += [os.path.dirname(__file__)]
```

Comment:
Why do we need this?

Reply:
not sure what is going on with rules_python - last few versions have all displayed this same issue that you cant import a .py file in the same folder without adding it to `sys.path`

this has held us back from upgrading for a long time now (maybe 6 months), i havent found a ticket for it, and havent found much point posting tickets previously, so i gave up and just did manually what rules_python should be doing (and did)

Reply:
> ... import a .py file in the same folder without adding it ...

fwiw adding a folder works and i think adding an `__init__.py` might also but i think this is the cleanest workaround i could think of

Reply:
Maybe file an issue upstream and annotate these sites? I don't mind unblocking, but it would be good to have some history here and way to unwind this.